src/qs_linres_module.F

```diff
@@ -240,7 +240,12 @@ SUBROUTINE linres_calculation_low(qs_env)
       lr_section => section_vals_get_subs_vals(qs_env%input, "PROPERTIES%LINRES")
       CALL section_vals_get(lr_section, explicit=lr_calculation)
 
-      IF (lr_calculation) THEN
+      ALLOCATE (linres_control) 
+      CALL section_vals_val_get(lr_section, "EVERY_N_STEP", i_val=linres_control%every_n_step)
```

Comment:
You can simply read the value into a local variable because only need it in the next line.
Please remove the allocate/DEALLOCATE of `linres_control`.

Reply:
Thanks. Done.